_posts/en/2019-02-04-rbf-in-the-wild.md

```diff
@@ -0,0 +1,408 @@
+---
+title: 'RBF in the Wild'
+permalink: /en/rbf-in-the-wild/
+name: 2019-02-04-rbf-in-the-wild
+type: posts
+layout: post
+lang: en
+version: 1
+---
+
+Opt-in Replace by Fee (RBF) was standardized in December 2015. But who is
+supporting it and what is the user experience like? This post is a study of
+Bitcoin wallet and block explorer support and usability of opt-in RBF
+([BIP125](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki)). The
+findings were initially presented during the most recent [Bitcoin Optech
+Workshop in Paris](/workshops/).
+
+Understanding how other wallets, exchanges, and block explorers handle opt-in
```

Comment:
In my fixup commit: I've changed 'Opt-in' -> 'opt-in' here and throughout the article. I don't see opt-in capitalized in the BIP or elsewhere so I think lower case is preferred.

Reply:
You’re right it should be lowercase.